apps/src/templates/teacherNavigation/teacher-navigation.module.scss

```diff
@@ -0,0 +1,21 @@
+@import 'color.scss';
+
+.sidebarContainer {
+  min-width: 242px;
+  max-width: 242px;
+  background-color: $light_gray_50;
+}
+
+.sidebarContent {
+  padding-top: 20px;
+  padding-left: 16px;
+  position: sticky;
+  top: 0;
+  width: 210px;
+}
+
+.sectionHeader {
+  margin-top: 0;
+  margin-bottom: 2px;
+  // TO DO: modify color of text after hearing from Molly
```

Comment:
This was intentionally left in.  This was also documented in the ticket as well.

Reply:
Could you link the ticket from the comment?

Reply:
To clarify, this was a documented within the ticket for this task.  It should be fixed before we move this ticket into the "done" column.  There was no separate ticket created for this change.

Reply:
More info - this is a result of the DSCO team making updates to how colors are used in the new design system. I put more info in Slack (teacher-dashboard-lounge).